<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt0515Class</class>
 <widget class="QMainWindow" name="qt0515Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt0515</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(220, 220, 220)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>720</y>
       <width>661</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ff0000;&quot;&gt;Image Crop QT&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Young-Mook Kang&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>581</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>C:/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>581</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>C:/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Output :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Input :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Crop Images</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>640</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Original Images</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>640</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>520</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>520</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>70</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>340</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>460</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>640</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>110</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>640</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>160</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>280</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>160</y>
       <width>280</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background : rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>460</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>460</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>460</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>460</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>500</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>280</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>645</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>645</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>140</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>460</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>80</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>640</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(198, 198, 198)</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>45</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>pushButton</zorder>
    <zorder>listWidget</zorder>
    <zorder>textBrowser</zorder>
    <zorder>label_3</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label_4</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>label_23</zorder>
    <zorder>label_22</zorder>
    <zorder>label_15</zorder>
    <zorder>label_24</zorder>
    <zorder>label_13</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>label_8</zorder>
    <zorder>label_10</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_11</zorder>
    <zorder>label_16</zorder>
    <zorder>label_9</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>label_21</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton_10</zorder>
    <zorder>label_6</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>label_7</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>label_19</zorder>
    <zorder>label_17</zorder>
    <zorder>label_20</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>label_12</zorder>
    <zorder>spinBox</zorder>
    <zorder>label_18</zorder>
    <zorder>pushButton_9</zorder>
    <zorder>label_14</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt0515.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>selectInputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>827</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>selectOutputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>slotDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>nextImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveCrop45()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveCrop70()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1067</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>1031</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveCrop80()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveCrop100()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1077</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>1015</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveCrop110()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>771</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>loadUserCrop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1022</x>
     <y>776</y>
    </hint>
    <hint type="destinationlabel">
     <x>962</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>qt0515Class</receiver>
   <slot>saveUserCrop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>1045</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotDisplay()</slot>
  <slot>selectInputDir()</slot>
  <slot>selectOutputDir()</slot>
  <slot>nextImage()</slot>
  <slot>saveCrop45()</slot>
  <slot>saveCrop70()</slot>
  <slot>saveCrop80()</slot>
  <slot>saveCrop100()</slot>
  <slot>saveCrop110()</slot>
  <slot>loadUserCrop()</slot>
  <slot>saveUserCrop()</slot>
 </slots>
</ui>
